src/less/navbar.less

```diff
@@ -17,7 +17,7 @@
   &:extend(.clearfix all);
 
   @media (min-width: @grid-float-breakpoint) {
-    border-radius: @navbar-border-radius;
+    .border-radius(@navbar-border-radius);
```

Comment:
What do you guys think about using Autoprefixer instead of defining all these mixins for vendor prefixing? We could easily hook up https://github.com/sindresorhus/gulp-autoprefixer to the build pipeline and simplify things a bit.